src/SourceGenerators/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <!-- Extend Directory.Build.* from parent directory -->
+  <Import Project="..\$(MSBuildThisFile)" />
+
+  <!-- Common properties for all source generation projects -->
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
I'm not a fan of generalizing things like this. Doesn't save much typing and makes things harder to follow for newcomers (who won't find the TF and get confused). I'd remove this and restore the property on each project

Reply:
More than the TargetFramework property itself, I was generalizing the _requisite_ that source generators target .NET Standard 2.0. I'll admit I should have made this clearer by adding a comment in Directory.Build.props pointing to relevant documentation.

Anyway, your point about newcomers still stands. Being a fan of the principle of least astonishment myself, I moved TargetFramework back into each project as suggested.